src/renderer/components/dock/create-resource.scss

```diff
@@ -1,2 +1,15 @@
 .CreateResource {
-}
\ No newline at end of file
+  .Select {
+    .Select__single-value {
+      width: 100%
+    }
+    &.TemplateSelect {
+      min-width: 190px;
+    }
+  }
+}
+
+.select-template-badge {
+  float: right;
+  vertical-align: middle;
+}
```

Comment:
Please use flexbox for badge positioning. Currently, there is a rendering bug
<img width="226" alt="floated badge" src="https://user-images.githubusercontent.com/9607060/111601777-bfcad880-87e3-11eb-8b67-ca0a3d15e244.png">

You can reference to `NamespaceSelectFilter` to look for an example (`Badge` can be placed instead of tick icon). `src/renderer/components/+namespaces/namespace-select-filter.tsx`